src/main/kotlin/com/example/olaf/blogtutorial/Extensions.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.olaf.blogtutorial
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatterBuilder
+import java.time.temporal.ChronoField
+
+
+fun LocalDateTime.format() = this.format(englishDateFormatter)
+
+private val daysLookup = (1..31).associate { it.toLong() to getOrdinal(it) }
+
+private val englishDateFormatter = DateTimeFormatterBuilder()
+    .appendPattern("yyyy-MM-dd")
+    .appendLiteral(" ")
+    .appendText(ChronoField.DAY_OF_MONTH, daysLookup)
+
+private fun getOrdinal(n: Int) = when {
+    n in 11..13 -> "${n}th"
+    n % 10 == 1 -> "${n}st"
+    n % 10 == 2 -> "${n}nd"
+    n % 10 == 3 -> "${n}rd"
+    else -> "${n}th"
```

Comment:
```suggestion
    n in 4..31 -> "${n}th"
```
else 보다는 이게 좋을 것 같아요.
날짜로 사용할 수 있는 값이 (1..31) 요것 뿐이고 그것으로 map 을 만든것이기 때문에 저 값뿐이 안들어오니깐요.

when 안에서 11..13 이 중복이라서 좀 애매하긴하네요 ㅋㅋ